tests/algorithms/test_bounded_standard_deviation.py

```diff
@@ -0,0 +1,10 @@
+import pydp as dp
+
+
+class TestBoundedStandardDeviation:
+    def test_c_api(self):
```

Comment:
Can we add a test on whether SD obtained is within a particular limit/ bound?

Reply:
Sure! I missed. I also improved the test readability.